packages/richtext-lexical/src/exports/react/components/RichText/index.tsx

```diff
@@ -65,18 +70,21 @@ export const RichText: React.FC<RichTextProps> = ({
     finalConverters = defaultJSXConverters
   }
 
-  return (
-    <div className={className ?? 'payload-richtext'}>
-      {editorState &&
-        !Array.isArray(editorState) &&
-        typeof editorState === 'object' &&
-        'root' in editorState &&
-        convertLexicalToJSX({
-          converters: finalConverters,
-          data: editorState,
-          disableIndent,
-          disableTextAlign,
-        })}
-    </div>
-  )
+  const content =
+    editorState &&
+    !Array.isArray(editorState) &&
+    typeof editorState === 'object' &&
+    'root' in editorState &&
+    convertLexicalToJSX({
+      converters: finalConverters,
+      data: editorState,
+      disableIndent,
+      disableTextAlign,
+    })
+
+  if (disableContainer) {
+    return content
```

Comment:
I think the return here should still be within a react fragment `<>{content}</>` so that the return of the function remains similar